Sources/TensorFlow/Loss.swift

```diff
@@ -36,6 +36,18 @@ public func meanAbsoluteError<Scalar: TensorFlowFloatingPoint>(
     return abs(expected - predicted).mean()
 }
 
+/// Computes the hinge loss between predictions and expectations.
```

Comment:
```suggestion
/// Returns the hinge loss between predictions and expectations.
```

#160 has some more context.